From 193919de7fd000c5459d8976ad3102f9ffd8fcbf Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Fri, 18 Aug 2017 09:49:36 +0800
Subject: [PATCH 2394/5242] MLK-16123 video: fbdev: dcss: change a print level
 to be debug

commit  a40cc07820819a2547c389857096c072905beff4 from
https://source.codeaurora.org/external/imx/linux-imx.git

Change a 'dev_info' call to 'dev_dbg' call to avoid
possible too many kernel messages printed out.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/video/fbdev/mxc/imx_dcss.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/fbdev/mxc/imx_dcss.c b/drivers/video/fbdev/mxc/imx_dcss.c
index 882827a..fed2fef 100644
--- a/drivers/video/fbdev/mxc/imx_dcss.c
+++ b/drivers/video/fbdev/mxc/imx_dcss.c
@@ -2678,7 +2678,7 @@ static int config_channel_pipe(struct dcss_channel_info *cinfo)
 
 	fb_node = fbi->node;
 
-	dev_info(&cinfo->pdev->dev, "begin config pipe %d\n", fb_node);
+	dev_dbg(&cinfo->pdev->dev, "begin config pipe %d\n", fb_node);
 
 	/* configure all the sub modules on one channel:
 	 * 1. DEC400D/DTRC
-- 
1.7.9.5

